### ISSUE TITLE
SIPUNCULA- Problematic taxa in Annelida

### ISSUE THREAD

**Issue by camiplata** [CatalogueOfLife/checklistbank#1270]
Sipuncula in COL is a [phylum](https://www.dev.checklistbank.org/dataset/3/taxon/0c01ef3f-c0b9-4739-bfec-b4719505f77f), in Worms a [order](https://www.dev.checklistbank.org/dataset/2011/taxon/urn%3Alsid%3Amarinespecies.org%3Ataxname%3A1268), in BOLD a [class](https://www.dev.checklistbank.org/dataset/37384/taxon/http%3A%2F%2Fbins.boldsystems.org%2Findex.php%2FTaxbrowser_Taxonpage%3Ftaxid%3D15).

In a [project](https://www.dev.checklistbank.org/catalogue/10026/assembly?assemblyTaxonKey=76c93776-a3d9-4603-ac15-23a4c6bb6d6e) merging COL, Worms, BOLD this creates a duplication of this taxon

<img width="567" alt="Captura de pantalla 2023-08-22 a la(s) 6 48 40 p m" src="https://github.com/CatalogueOfLife/checklistbank/assets/5700527/a8d94b2b-c3c4-457d-9955-3b347b988eb7">

This is a potential editorial decision at the xrelease, or do we have an "automated" way to handle this on the xrelease? for example by using the priorities in this case worms over BOLD to set SIPUNCULA as an order

This is also  shows a larger data problem in the current COL list, in a quick search it looks like SIPUNCULA as [phylum is  very outdated](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2667193) (this comes from ITIS) . I see worms is currently working on a Annelida subregistry but this probably will not cover all Annelida: 
<img width="619" alt="Captura de pantalla 2023-08-22 a la(s) 6 44 54 p m" src="https://github.com/CatalogueOfLife/checklistbank/assets/5700527/0f676f08-49ea-4c02-b39a-0bb17850e024">

I think this may be a priority area for COL, there is also a printed catalogue from 2022 about Annelida that may help with this issue: https://global.oup.com/academic/product/annelida-9780199692309?cc=co&lang=en&

**Comment by mdoering** [comment]
Good point. Keeping a clean hierarchy is probably the most difficult part of integrating heterogeneous taxonomies.
When merging datasets by default we exclude any taxa above family rank for this very reason. If there is need to fill gaps in the COL taxonomy on higher levels this would be better handled directly in the main COL project - potentially even via small subprojects just doing providing a classification for a certain group down to superfamily or family level.

Even if higher ranks in merge sources are excluded, they still bear meaning as they are used to "snap" the parent of inserted taxa to known taxa in the project/release. So good matching does still matter in the case of Sipuncula and currently the phylum, class or order will be treated potentially as different taxa. It would be good to test how the merge works when higher ranks are excluded.
